src/connections/index.ts

```diff
@@ -1,5 +1,11 @@
+export const API_URL =
```

Comment:
Should we put this within `./connections/outerbase.ts` instead?

Reply:
Done